src/Mvc/samples/MvcSandbox/Pages/Components.cshtml

```diff
@@ -15,8 +14,8 @@
     <link href="css/site.css" rel="stylesheet" />
 </head>
 <body>
-    <app>@(await Html.RenderComponentAsync<MvcSandbox.Components.App>(RenderMode.Static))</app>
+    <component type="typeof(MvcSandbox.Components.App)"render-mode="Static" />
 
-    <script src="_framework/components.server.js"></script>
+    <script src="~/_framework/blazor.server.js"></script>
```

Comment:
Why add the `~/`? As long as the base href is set correctly, it will work without it. And if the base href isn't set correctly, other things will be broken.

Reply:
I was messing with something and this was accidental. I'll revert this 👍